processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/branch_hazard.asm (1 of 1)
processing file asmFiles/aluop_and_alubranch.asm (1 of 36)
processing file asmFiles/beq_notTaken.asm (2 of 36)
processing file asmFiles/beq_taken.asm (3 of 36)
processing file asmFiles/bge_notTaken.asm (4 of 36)
processing file asmFiles/bge_taken.asm (5 of 36)
processing file asmFiles/bgeu_notTaken.asm (6 of 36)
processing file asmFiles/bgeu_taken.asm (7 of 36)
processing file asmFiles/blt_notTaken.asm (8 of 36)
processing file asmFiles/blt_taken.asm (9 of 36)
processing file asmFiles/bltu_notTaken.asm (10 of 36)
processing file asmFiles/bltu.taken.asm (11 of 36)
processing file asmFiles/bne_notTaken.asm (12 of 36)
processing file asmFiles/bne_taken.asm (13 of 36)
processing file asmFiles/branch_hazard2.asm (14 of 36)
processing file asmFiles/branch_hazard.asm (15 of 36)
processing file asmFiles/count_days.asm (16 of 36)
processing file asmFiles/fetch_and_instr.asm (17 of 36)
processing file asmFiles/fib.asm (18 of 36)
processing file asmFiles/halt.asm (19 of 36)
processing file asmFiles/jal.asm (20 of 36)
processing file asmFiles/jalr.asm (21 of 36)
processing file asmFiles/mergesort.asm (22 of 36)
processing file asmFiles/mult.asm (23 of 36)
processing file asmFiles/mult_procedure.asm (24 of 36)
processing file asmFiles/other.asm (25 of 36)
processing file asmFiles/read_after_write.asm (26 of 36)
processing file asmFiles/search.asm (27 of 36)
processing file asmFiles/temptest.asm (28 of 36)
processing file asmFiles/test.branchjump1.asm (29 of 36)
processing file asmFiles/test.ldst1.asm (30 of 36)
processing file asmFiles/test.ldst2.asm (31 of 36)
processing file asmFiles/test.loadstore.asm (32 of 36)
processing file asmFiles/test.loaduse.asm (33 of 36)
processing file asmFiles/test.rtype.asm (34 of 36)
processing file asmFiles/write_after_read.asm (35 of 36)
processing file asmFiles/write_after_write.asm (36 of 36)
processing file asmFiles/mult.asm (1 of 1)
--- memsim.hex	2026-10-03 17:44:04.697111010 -0400
+++ memfpga.hex	2026-10-03 17:44:04.685111145 -0400
@@ -21,6 +21,4 @@
 :04001400FFC1011314
 :040015000081202323
 :0400160000008067FF
-:043FFD0000000004BC
-:043FFE000000000CB3
 :00000001FF
processing file asmFiles/temptest.asm (1 of 1)
--- memsim.hex	2026-10-03 17:58:31.252335742 -0400
+++ memfpga.hex	2026-10-03 17:58:31.240335878 -0400
@@ -15,5 +15,4 @@
 :04000E0000000013DB
 :04000F0000000013DA
 :04001000FFFFFFFFF0
-:0400240000000006D2
 :00000001FF
